transcode/transcoder_caching.go

```diff
@@ -7,21 +7,24 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"sort"
 	"sync"
+	"time"
 )
 
 const perm = 0o644
 
 type CachingTranscoder struct {
 	cachePath  string
 	transcoder Transcoder
+	limitMb    int
 	locks      keyedMutex
 }
 
 var _ Transcoder = (*CachingTranscoder)(nil)
 
-func NewCachingTranscoder(t Transcoder, cachePath string) *CachingTranscoder {
-	return &CachingTranscoder{transcoder: t, cachePath: cachePath}
+func NewCachingTranscoder(t Transcoder, cachePath string, limitMb int) *CachingTranscoder {
```

Comment:
looks megabits to me. how about a big B for bytes

```suggestion
func NewCachingTranscoder(t Transcoder, cachePath string, limitMB int) *CachingTranscoder {
```

Reply:
Done.